packages/jest-util/src/__tests__/error_with_stack.test.js

```diff
@@ -0,0 +1,16 @@
+import ErrorWithStack from '../error_with_stack';
```

Comment:
would be cool to flow it up

Reply:
That's fine. It's just nice to have flow in tests too so you don't end up testing scenarios that are already covered/prevented by type checker.